src/myframe.h

```diff
@@ -695,13 +695,15 @@ struct MyFrame : wxFrame {
             idd = new ImageDropdown(tb, imagepath);
             tb->AddControl(idd);
             tb->Realize();
+            tb->Show(!sys->zenmode);
         }
 
         if (showsbar) {
             wxStatusBar *sb = CreateStatusBar(4);
             sb->SetOwnBackgroundColour(toolbgcol);
             SetStatusBarPane(0);
             SetDPIAwareStatusWidths();
+            sb->Show(!sys->zenmode);

```

Comment:
Rather than calling this "zenmode" which noone knows what exactly it will do, instead have 2 settings for status and toolbar on/off? That is way clearer and also means you can do one not the other

Reply:
Thanks for your feedback. I split the zen mode into two persistent options. This also makes the entries in the Options menu obsolete I think.